static/src/stylesheets/module/_adslot.scss

```diff
@@ -224,12 +224,25 @@
     }
 }
 .ad-slot--gallery-inline {
+    background-color: $media-background;
+
     @include mq(mobileLandscape) {
         width: 300px;
     }
     @include mq(tablet) {
         width: auto;
     }
+    @include mq(desktop) {
+        &,
```

Comment:
It keeps the ad slot label and the ad itself left aligned as all the images are at desktop breakpoint. You've made me realise that the following lines of css could be removed though!